### ISSUE TITLE
Stability issues since update to 0.1.9

### ISSUE THREAD

**Issue by suuronenh** [gazoodle/gecko-home-assistant#95]
I've seen several device/sensors unavailable situations after version was automatically updated to 0.1.9 two days ago. Integration had been rock solid before this. No other explicit changes have been applied in the same time frame. Reloading the integration resolves the situation but within few hours the same problem occurs. At the same time Gecko mobile app can successfully access the information. I have tried power cycling the gecko device with no effect.

I can provide additional information, just tell what is needed.

I also tried to revert the version to 0.1.8 following this guide: https://hacs.xyz/docs/faq/select_version/. But after downloading 0.1.8 and then restarting HASS, the version was still reported as 0.1.9.

**Comment by suuronenh** [comment]
After updating HA core/OS and HACS the issue has not repeated itself. Versions were very recent, only single minor version difference. So the solution might have been just restarting HASS. But my HASS is automatically restarted every night and previous restarts did not solve the issue.

I did take a look on the release notes and the only thing catching my eye was related to installing older versions of plugins within HACS. There was a fix related to non-zip repositories. I tried installing the previous 0.1.8 version with the latest HACS but I could not even proceed to downloading. After selecting 0.1.8 from the dropdown it kept reverting to 0.1.9 without any notifications.

There is no need to keep this open if no one else has encountered similar issues with 0.1.9. Perhaps the ability to install older versions could be a feature request (if it is up to the plugin to support the mechanism).
